codemod/src/plugin-deprecate/plugin-deprecate-props.ts

```diff
@@ -33,7 +40,7 @@ export default function (): PluginObj<Context> {
                 if (
                   t.isImportSpecifier(specifier) &&
                   t.isIdentifier(specifier.imported) &&
-                  isComponentDeprecated(specifier.imported.name)
+                  Boolean(this.deprecations[specifier.imported.name])
```

Comment:
This is optionally chained in deprecate-atoms, should it be the same here too?
```suggestion
                  Boolean(this.deprecations?.[specifier.imported.name])
```

Reply:
I'll go the other way, deprecations are required, i'll remove the optional chain (and defaulting to blank object) from the other locations.